eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -849,8 +849,11 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
         }
       }
 
-    case Event(c@CurrentBlockCount(count), d: DATA_NORMAL) if d.commitments.timedoutOutgoingHtlcs(count).nonEmpty =>
-      handleLocalError(HtlcTimedout(d.channelId, d.commitments.timedoutOutgoingHtlcs(count)), d, Some(c))
+    case Event(c@CurrentBlockCount(count), d: DATA_NORMAL) if d.commitments.timedOutOutgoingHtlcs(count).nonEmpty =>
```

Comment:
It's a bit redundant to have these two cases duplicated in 4 states...
Shall I refactor that? Maybe by creating a partial receive that handles these two cases and re-using it in the 4 states that need it?